src/components/app.jsx

```diff
@@ -80,23 +70,49 @@ const App = React.createClass({
                 <div className="col-md-9">
                   <p id="baseline">
                     <span className="message">
-                      <FormattedMessage {...messages.header} />
+                      <FormattedMessage id = "header"/>
+                      <small>
+                        <ExternalLink href = "https://www.openfisca.fr" target = "_blank">
+                          <strong><FormattedMessage id = "learnMore"/></strong>
```

Comment:
@fpagnoux It could be as it is more semantic (I would argue for `class="metadata"` which is less specific). I am lacking context on that case, though. But in the current case, where there is a native selector that does make sense (we want to increase the font weight of the `header a` because links show up in a lighter colour that has to be compensated only because they are in the header, which has a lighter font-weight), it would be better to use CSS than markup. There is nothing semantic in the `<strong>`: the meaning is already, and better, carried by the `<a>`.